.github/workflows/build.yml

```
@@ -23,5 +23,6 @@ jobs:
         # Until we have arm64 runners, we can't automatically test arm64 wheels
         - cp3*-macosx_arm64
       sdist: true
+      test_command: python -c "from stcal.ramp_fitting.ols_cas22 import _ramp, _jump, _fit"
```

add the following (but only after merging #156):
```suggestion
      test_command: python -c "from stcal.ramp_fitting.ols_cas22 import _ramp, _jump, _fit; from stcal.ramp_fitting import slope_fitter"
```